includes/insights-session-health-frontend.mdx

```diff
@@ -0,0 +1,30 @@
+## Charts
+
+![Insights Frontend Session Health Tab](./common-imgs/img/frontend-insights-session-health.png)
+
+For the Frontend module, the following charts are available:
+
+- Unhealthy Sessions
+- Session Health
+- Session Counts
+- User Health
+- User Counts
+- [Issues](/product/issues/)
+
+Most of these charts, with the exception of the Issues chart, are based on session-backed data.
+
+The Session Health, Session Counts, User Health, and User Counts charts aim to break down trends in how your sessions are performing status-wise. The status options are healthy, crashed, errored, or abnormal. [Learn more about session statuses](/product/releases/health/#session-status).
+
+The Unhealthy Sessions chart combines the total rate of errored, crashed, and abnormal sessions into one line.
+
+The Issues chart is the only chart that does not rely on session-backed data. Instead, it shows the number of new and resolved issues for that project over time. It also gives a preview of the two most recently seen issues for that project. The chart header, when hovered, contains a link to view all issues for that project.
```

Comment:
```suggestion
The Issues chart shows the number of new and resolved issues for that project over time. It also gives a preview of the two most recently seen issues for that project. The chart header, when hovered, contains a link to view all issues for that project.
```

Reply:
I don't think we need to emphasize the fact that it's not sessions-backed

Reply:
> The chart header, when hovered, contains a link to view all issues for that project.

It took me a few minutes to understand this. I only noticed now that there's a "View All" button when hovering.

Reply:
Maybe we can update to..

> The chart, when hovered, has a button to view all issues for that project.